vital-religion/introduction.adoc

```diff
@@ -1,30 +1,29 @@
 == Introduction.
 
-When we look around us,
+When we look around us
 and observe how many conflicting creeds and systems of
-religion are sedulously propagated in the present day,
+Christianity are continually promoted in the present day,
```

Comment:
I used "actively" here because it seemed like sedulously meant "energetically" or "busily"